src/main/resources/view/GuestListCard.fxml

```diff
@@ -27,9 +26,7 @@
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
-      <FlowPane fx:id="tags" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
       <Label fx:id="numberOfGuests" styleClass="cell_small_label" text="\$numberOfGuests" />
       <Label fx:id="dateRange" styleClass="cell_small_label" text="\$dateRange" />
```

Comment:
Should dateRange be above numberOfGuests for consistency?

Reply:
Agree as well, fixed in latest commit!